source/renderer/app/stores/WalletsStore.js

```diff
@@ -613,12 +624,23 @@ export default class WalletsStore extends Store {
       });
 
       if (wallet) {
+        logger.debug(
+          '[HW-DEBUG] HWStore - Execute HW create / restore - Patch HW'
+        );
         await this._patchWalletRequestWithNewWallet(wallet);
+        logger.debug('[HW-DEBUG] HWStore - GO TO ROUTE');
         this.goToWalletRoute(wallet.id);
+        logger.debug('[HW-DEBUG] HWStore - Refresh wallets data');
         this.refreshWalletsData();
+        logger.debug('[HW-DEBUG] HWStore - Close dialog');
         this.actions.dialogs.closeActiveDialog.trigger();
       }
+      this._resumePolling();
     } catch (error) {
+      logger.debug('[HW-DEBUG] HWStore - HW create / restore ERROR', {
+        error,
+      });
+      this._resumePolling();
```

Comment:
@tomislavhoracek please move "this._resumePolling();" from both try and catch blocks into a finally block 🙏

Reply:
@nikolaglumac Done!